clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/loadbalancing/GoogleInternalHttpLoadBalancer.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.google.model.loadbalancing;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.netflix.spinnaker.clouddriver.model.LoadBalancerServerGroup;
+import groovy.transform.ToString;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+
+@ToString(includeSuper = true)
+@groovy.transform.EqualsAndHashCode(callSuper = true)
```

Comment:
You can replace these with lombok:

```suggestion
@Data
@EqualsAndHashCode(callSuper = true)
@ToString(callSuper = true)
```

Then:
1) mark `type` and `loadBalancingScheme` as final
2) delete all the getters and setters (except `getView`)

Reply:
Thank you for detailed instruction! It's very helpful.